media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -1565,6 +1566,33 @@ void DestroyMediaContextMutex(PDDI_MEDIA_CONTEXT mediaCtx)
     return;
 }
 
+static int DdiMedia_SelectIntelDevice()
+{
+    int intel_gpu_index = -1;
+    for (int i = 0; i < 16; ++i) {
```

Comment:
why not use drmGetDevices2 to get the device list?

Reply:
Very good suggestion! Here the functionality is nearly the same. I will consider use that API in the future.

In our specific case, we don't need to open all devices. Rather, we can stop early when the target device is found.